README.md

```diff
@@ -126,3 +126,15 @@ Django Settings
 AssertRegex
 * From [StackOverflow](https://stackoverflow.com/a/64192098)
 * Accessed Feb 14 2021
+
+Django Restframework
+* From [Bennett Garner](https://medium.com/swlh/build-your-first-rest-api-with-django-rest-framework-e394e39a482c)
+* Accessed Feb 15 2021
+
+Fix Django Authenticate Token issue 
+* From [ilse2005](https://stackoverflow.com/questions/35601130/django-rest-framework-using-tokenauthentication-with-browsable-api)
```

Comment:
Similar to comment above, seperate the stack overflow user from the actual page you got this from